puppet/modules/slave/manifests/unittests.pp

```diff
@@ -168,4 +168,9 @@
 
   # RVM
   include slave::rvm
```

Comment:
Perhaps put this in the `else` below so you either get rvm or rbenv? Perhaps even go further and introduce a parameter that's`$rbenv = $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '9'`